### ISSUE TITLE
403 Forbidden

### ISSUE THREAD

**Issue by EhsanFox** [DevSnowflake/soundcloud-scraper#27]
For some reason, calling `getSongInfo` throws the error

I haven't investigated it, but it either is a problem with the CLIENT_ID (which I don't pass to the class/function)

Or it can be perhaps a moved endpoint or some similar action by Soundcloud.

P.S: `fetchStreamURL` is set to `true` (which obviously calls the function `fetchSongStreamURL`

```
Error: [Code: 403] Rejected with status code "403"!
    at /workspaces/backend/node_modules/soundcloud-scraper/src/util/Util.js:161:35
```

**Comment by twlite** [comment]
Yes, based on the status code, the error seems to be caused by client id being invalid

**Comment by twlite** [comment]
what does `await Util.keygen(true)` return?

**Comment by EhsanFox** [comment]
I've never called it or passed it to the Client, I've just left that empty so the package can generate one,
I can try and create one and pass it on to the client as well as log it to see what's the content if you want to.

**Comment by twlite** [comment]
v6 rewrite initiated https://github.com/skdhg/soundcloud-scraper/pull/28, although I might not be able to complete it in time. Contributions are welcome!
